index.bs

```diff
@@ -2014,6 +2014,132 @@ whose cues all have a <a>cue payload</a> that is <a>WebVTT chapter title text</a
 said to be a <dfn>WebVTT file using cue text</dfn>.</p>
 
 
+<h2 id=css-extensions-auth>CSS extensions</h2>
+
+<p><i>This section is non-normative.</i></p>
+
+<p class="note">This section lists new CSS pseudo-elements and pseudo-selectors that are made
+available to CSS user agents to allow authors to manipulate the visual rendering of cue content.
+These features are quite tightly linked to media elements in HTML. When supporting WebVTT in
+media players that don't support CSS, equivalent visual rendering control will need to be
+made available.</p>
+
+
+<h3 id=css-extension-cue>The ''::cue'' pseudo-element</h3>
+
+<p>The ''::cue'' pseudo-element describes the content of all cues of a video element.</p>
+
+<pre>
+CSS example:
+
+video.upper ::cue { text-transform: uppercase }
```

Comment:
'text-transform' is not a safelisted property, it does not apply for WebVTT. Maybe use 'color' instead?